src/tests/ImportNameRuleTests.ts

```diff
@@ -146,7 +146,7 @@ describe('importNameRule', () : void => {
             import baseChartOptions = require('common/component/chart/options/BaseChartOptions');
         `;
 
-        const options: any[] = [ true, {
+        const options = [ true, {
```

Comment:
I cheated slightly here and let the type of `options` be inferred. It sort of seems like it goes against the conventions, but it didn't break any lint rules. 🤷‍♂️